wpimath/src/main/native/include/frc/estimator/PoseEstimator.inc

```diff
@@ -71,7 +71,7 @@ void PoseEstimator<WheelSpeeds, WheelPositions>::ResetTranslation(
 template <typename WheelSpeeds, typename WheelPositions>
 void PoseEstimator<WheelSpeeds, WheelPositions>::ResetRotation(
     const Rotation2d& rotation) {
-  m_odometry.ResetTranslation(rotation);
```

Comment:
Huh? How did this even compile in the first place?

Reply:
> Huh? How did this even compile in the first place?

The templates don't get instatiated in the build process unless they are used somewhere.

Reply:
Which means there isn't test coverage for this function